packages/rexray/buildinfo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "docker": "golang:1.6",
+  "single_source" : {
+    "kind": "git",
+    "git": "git@github.com:dcos/rexray.git",
+    "ref": "1816ee3868550e07b90f00f57bfd7f737235cd93",
+    "ref_origin": "v0.3.3-dcos"
```

Comment:
@jdef should we hold off on publishing the dvdcli tarball to our downloads bucket until we know more about the regression?